src/unix/bsd/freebsdlike/mod.rs

```diff
@@ -1572,26 +1572,10 @@ extern "C" {
     pub fn getutxid(ut: *const utmpx) -> *mut utmpx;
     pub fn getutxline(ut: *const utmpx) -> *mut utmpx;
     pub fn initgroups(name: *const c_char, basegid: crate::gid_t) -> c_int;
-    #[cfg_attr(
-        all(target_os = "freebsd", any(freebsd11, freebsd10)),
-        link_name = "kevent@FBSD_1.0"
-    )]
-    pub fn kevent(
```

Comment:
Did you really mean to remove this function?  I think you should just remove the `link_name` instead.  The function is definitely needed.